includes/definitions/benu.yaml

```diff
@@ -0,0 +1,21 @@
+os: benu
+group: Benu
+text: 'Benu'
+type: network
+icon: benu
+mib_dir:
+    - benu
+discovery:
+    - sysDescr:
+        - BenuOS
```

Comment:
The OID you have in your test `.1.3.6.1.4.1.39406.1.2.1.1` so you could do something like:

```
    - sysObjectId:
        - 1.3.6.1.4.1.39406.1.2
```

Instead of the sysDescr check you have